install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -11,18 +11,20 @@ metadata:
     release: {{ .Release.Name }}
 data:
   mesh: |-
+    {{- if .Values.mixer.policy.enabled }}
     # Set the following variable to true to disable policy checks by the Mixer.
     # Note that metrics will still be reported to the Mixer.
     disablePolicyChecks: {{ .Values.global.disablePolicyChecks }}
+    {{- end }}
```

Comment:
@douglas-reid is there any value to disable mixer check explicitly when policy is not enabled?

Reply:
@linsun do you mind clarifying the question?  are you asking: "if no policies are configured, should calling `istio-policy` be auto-disabled?" If so, then I think that is a desired state -- just not one that is supported *yet*.

Reply:
@douglas-reid  Nothing magic behavior here. :).  I was just trying to see if we should configure explicitly `disablePolicyChecks: false` in configmap when policy isn't enabled.  If there is no value, @jwendell 's update LGTM.